confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java

```diff
@@ -727,8 +736,18 @@ private TSStatus checkConfigNodeGlobalConfig(TConfigNodeRegisterReq req) {
 
   @Override
   public TSStatus createPeerForConsensusGroup(List<TConfigNodeLocation> configNodeLocations) {
-    consensusManager.createPeerForConsensusGroup(configNodeLocations);
-    return StatusUtils.OK;
+    for (int i = 0; i < 30; i++) {
+      try {
+        if (consensusManager == null) {
+          Thread.sleep(1000);
+        } else {
+          consensusManager.createPeerForConsensusGroup(configNodeLocations);
+          return StatusUtils.OK;
+        }
+      } catch (Exception ignored) {
+      }
```

Comment:
This process of Exception is too simply. From my perspective, this way may ignore some useful exception caused by createPeerForConsensusGroup, so I think you can use multi catch in level.